x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/exploration_results_table/use_exploration_results.ts

```diff
@@ -74,12 +75,16 @@ export const useExplorationResults = (
 
   const fetchColumnChartsData = async function () {
     try {
-      if (jobConfig !== undefined) {
-        const columnChartsData = await fetchChartsData(
-          jobConfig.dest.index,
-          ml.esSearch,
-          searchQuery,
-          columns.filter((cT) => dataGrid.visibleColumns.includes(cT.id))
+      if (jobConfig !== undefined && indexPattern !== undefined) {
+        const dataLoader = new DataLoader(indexPattern, toastNotifications);
```

Comment:
same as above, `dataLoader` should be initialized based on the `interPattern` changes with `useMemo`

Reply:
Changed in 3883477.